_data/sites.json

```diff
@@ -7027,17 +7027,16 @@
 
     {
         "name": "GNOME",
-        "url": "https://www.gnome.org/privacy-policy/",
-        "difficulty": "hard",
-        "email": "info@gnome.org",
-        "notes": "Account deletion requires contacting Customer Support.",
-        "notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
-        "notes_fr": "Pour supprimer votre compte contactez le support.",
-        "notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
-        "notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
-        "notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
-        "notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
-        "notes_es": "Para borrar tu cuenta contacta con asistencia al cliente",
+        "url": "https://extensions.gnome.org/accounts/settings",
+        "difficulty": "medium",
+        "notes": "In the section Account removal, you need to accept the option delete your account, and input your password.",
+        "notes_tr": "Hesap kaldırma bölümünde, hesabınızı silme seçeneğini kabul etmeniz ve şifrenizi girmeniz gerekiyor.",
+        "notes_fr": "Dans la section Suppression de compte, vous devez accepter l'option supprimer votre compte et saisir votre mot de passe.",
+        "notes_it": "Nella sezione Rimozione account, è necessario accettare l'opzione elimina il tuo account e inserire la tua password.",
+        "notes_de": "Im Abschnitt Konto entfernen müssen Sie die Option Ihr Konto löschen akzeptieren und Ihr Passwort eingeben.",
+        "notes_pt_br": "Na seção Remoção de conta, você precisa aceitar a opção de excluir sua conta e inserir sua senha.",
+        "notes_cat": "A la secció Eliminació de compte, cal que accepteu l'opció d'eliminar el vostre compte i introduir la vostra contrasenya.",
+        "notes_es": "En la sección Eliminación de cuenta, debes aceptar la opción de eliminar tu cuenta e ingresar tu contraseña.",
```

Comment:
There is no need to provide translations for all languages, unless you speak them all. If  you don't, we ask you to please leave only the ones you're fluent.